iroha-cpp/services/CMakeLists.txt

```
@@ -0,0 +1,20 @@
+add_library(clientservices
```

IMO better name would be:
```suggestion
add_library(iroha_cpp_client_services
```
Because those services are only for iroha library